src/main/java/website/frontrow/board/Bubble.java

```diff
@@ -157,4 +175,50 @@ public void addToLog(String action)
     {
         Log.add(action);
     }
+    
+    /**
+     * Getter for timeEmpty.
+     * @return timeEmpty long
+     */
+    public long getTimeEmpty()
+    {
+    	return timeEmpty;
+    }
+    
+    /**
+     * Setter for timeEmpty.
+     * @param time long
+     */
+    public void setTimeEmpty(long time)
+    {
+    	timeEmpty = time;
+    }
+    
+    /**
+     * Makes the bubble float upwards at the right time.
+     */
+    public void floatUpwards()
+    {
+    	timeEmpty++;
+    	
+    	if(timeEmpty >= TIME_FLOAT_UPWARDS)
+    	{
+    		this.hit();
+    		addToLog("[BUBBLE]\t is empty and floating upwards");
+    	}
+    }
+    
+    /**
+     * Kills the bubble at the right time.
+     */
+    public void killBubble()
+    {
+    	timeEmpty++;
```

Comment:
Then make it clear that this is the case, for example, make the later constant depend on the earlier constant. So that if the earlier one changes, the amount of time till the bubble pops does not change.